hack/clibyexample/template

```diff
@@ -0,0 +1,12 @@
+{{range .items}}
```

Comment:
Don't think so, I think it helps to fix the command specially given this is a "by example" doc. Also, notice that `name` will only return the current command name, so in `oc config set-credentials` it will only return `set-credentials`, for example.